recipes/thrust/all/conanfile.py

```diff
@@ -0,0 +1,39 @@
+from conans import ConanFile, tools
+import os
+
+class ThrustConan(ConanFile):
+    name = "thrust"
+    license = "Apache-2.0"
+    description = ("Thrust is a parallel algorithms library which resembles"
+                   "the C++ Standard Template Library (STL).")
+    topics = ("parallel", "stl", "header-only")
+    homepage = "https://thrust.github.io/"
+    url = "https://github.com/conan-io/conan-center-index"
+    no_copy_source = True
+    options = {"thrust_device_system": ["cuda", "cpp", "omp", "tbb"]}
```

Comment:
Better name would be just `device_system`, options are already scoped to the package. Having to write `-o thrust:thrust_device_system=xxxx`, I prefer just `-o thrust:device_system=xxxx`

Reply:
Agree